### PR TITLE
Add gradient border

### PR DESCRIPTION
**Pull request by upacyxou**
- gradient border added

### CODE REVIEW

File: src/components/ui/Card.tsx [hunk 1]
```diff
@@ -15,13 +15,18 @@ import {
...
+const cardContainer = (
+  small?: boolean,
+  blueBg?: boolean,
+  gradient?: boolean
+) => {
   const card = classnames(
     position('relative'),
     borderWidth(blueBg ? 'border-0' : 'border'),
-    borderColor('border-half-grey'),
+    borderColor({ 'border-half-grey': gradient }),
```
Comment (T-Damer) [inline src/components/ui/Card.tsx:29]: Should't there be `!gradient`?

File: src/components/ui/GradientBorder.tsx [hunk 2]
```diff
@@ -9,23 +9,35 @@ import classnames, {
...
-  boxShadow('shadow-2xl', 'hover:shadow-lg', 'active:shadow-button-active')
-)
-const innerButtonWrapper = classnames(
-  width('w-full'),
-  borderRadius('rounded-full'),
-  backgroundColor('bg-primary-dark')
-)
+const parentButtonWrapper = (smallRadius?: boolean, withoutShadow?: boolean) =>
+  classnames(
+    borderRadius(smallRadius ? 'rounded-2xl' : 'rounded-full'),
```
Comment (T-Damer) [inline src/components/ui/GradientBorder.tsx:14]: How about doing this one time? Now you have same check in 2 functiona

**Review by T-Damer — APPROVED** [review]

File: src/components/ui/Card.tsx (hunk 1 above)
Comment (upacyxou) [reply src/components/ui/Card.tsx:29]: you're right, ty

File: src/components/ui/GradientBorder.tsx (hunk 2 above)
Comment (upacyxou) [reply src/components/ui/GradientBorder.tsx]: fixed, ty

**Review by MixailE — APPROVED** [review]

**Pull request merged**